packages/itwin/map-layers/package.json

```diff
@@ -38,6 +38,9 @@
   "dependencies": {
     "@itwin/itwinui-icons-color-react": "^2.1.0",
     "@itwin/itwinui-icons-react": "^2.6.0",
+    "@itwin/itwinui-react": "^2.11.2",
+    "@itwin/itwinui-variables": "^2.0.0",
+    "@itwin/map-layers-formats": "^4.1.2",
```

Comment:
why is this a direct dep? is this dep pure / has no side effects?
if its not, needs to be dev + peer dep